modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/OAuth2RestClient.java

```diff
@@ -187,6 +188,21 @@ public void updateInboundDetailsOfApplication(String appId, Object inboundConfig
         }
     }
 
+    /**
+     * Delete an Inbound Configuration
+     *
+     * @param appId Application id.
+     * @param inboundType Inbound Type to be deleted.
+     */
+    public Boolean deleteInboundConfiguration(String appId, String inboundType) throws IOException {
```

Comment:
In the method description add the @throws IOException <root/reason the exception from the POV of this method>. Let's keep this as a practice.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096